### PR TITLE
add option post_type to post-list directive

### PR DESCRIPTION
**Pull request by davidak**
all possibilities tested (with python 3.5)

### CODE REVIEW

File: nikola/plugins/compile/rest/post_list.py [hunk 1]
```diff
@@ -144,20 +149,22 @@ def run(self):
...
+        post_type = self.options.get('post_type')
         show_all = self.options.get('all', False)
         lang = self.options.get('lang', utils.LocaleBorg().current_lang)
         template = self.options.get('template', 'post_list_directive.tmpl')
         sort = self.options.get('sort')
 
-        output = _do_post_list(start, stop, reverse, tags, categories, slugs, show_all,
-                               lang, template, sort, state=self.state, site=self.site)
+        output = _do_post_list(start, stop, reverse, tags, categories, slugs, post_type,
+                    show_all, lang, template, sort, state=self.state, site=self.site)
```
Comment (Kwpolska) [inline nikola/plugins/compile/rest/post_list.py]: Fix the indentation so that you follow PEP 8 (`show_all` directly under `start`)

**Comment by davidak** [comment]
done.

`flake8 nikola/` returns no warnings.

**Comment by Kwpolska** [comment]
I’ll review it when I’ll have more time.

**Comment by ralsina** [comment]
LGTM

**Comment by ralsina** [comment]
Thanks for the PR @davidak !

**Pull request merged**